sdk/schemaregistry/azure-schemaregistry/tsp-location.yaml

```diff
@@ -0,0 +1,4 @@
+cleanup: false
+commit: 8c473d0d5890554bc6a4b2101b7dc26bfa734b41
```

Comment:
When you merge the tsp into main, you should update this commit to point to the merged commit

Reply:
will do!